src/appshell/qml/Preferences/internal/LanguagesSection.qml

```diff
@@ -74,7 +74,7 @@ BaseSection {
 
             anchors.verticalCenter: parent.verticalCenter
 
-            text: qsTrc("appshell", "Check for update")
+            text: qsTrc("appshell", "Check for language updates")
```

Comment:
Rather "translation updates"?

Reply:
And if I well understood, if I would like to change text only, should I modify only this or similar file?